.gitlab-ci.yml

```diff
@@ -80,8 +80,14 @@ configure-docker-image:
     - apt-get install -y git skopeo python3
     - |
       echo "Getting package version and checking whether we need to build the container"
-      PACKAGE_VERSION=$(git describe --tags)
-      REGISTRY_IMAGE_TAG=$(echo ${PACKAGE_VERSION%-*} | sed 's/-/.dev/')
+      if [ -n "${LIBSONATAREPORT_TAG}" || -n "${LIBSONATA_TAG}" || -n "${NEURON_COMMIT_ID}" && -z "${REGISTRY_IMAGE_TAG}" ]; then
```

Comment:
We need to define these 4 variables as the env `variables`, right?

Reply:
You can pass any of them as variables when manually running the pipeline. If you do, you have to include `REGISTRY_IMAGE_TAG` (so you'll be specifying at least two of them :-) )

Reply:
ah, yes, indeed.